packages/charting/src/components/HorizontalBarChart/HorizontalBarChart.types.ts

```diff
@@ -102,4 +108,26 @@ export interface IHorizontalBarChartStyles {
    * styles for hover card data
    */
   hoverCardDataStyles?: IStyle;
+
+  /**
+   * Style for the benchmark triangle
+   */
+  triangle: IStyle;
+}
+
+export enum ChartDataMode {
+  /**
+   * show the datapoint.x value
+   */
+  default = 0,
+
+  /**
+   * show the fraction of datapoint.x/datapoint.y
+   */
+  fraction = 1,
+
+  /**
+   * show the percentage of (datapoint.x/datapoint.y)%
+   */
+  percentage = 2
 }
```

Comment:
We generally prefer string unions over enums. [Here is a TypeScript Playground](https://www.typescriptlang.org/play/#src=export%20enum%20ChartDataMode%20%7B%0D%0A%20%20%2F**%0D%0A%20%20%20*%20show%20the%20datapoint.x%20value%0D%0A%20%20%20*%2F%0D%0A%20%20default%20%3D%200%2C%0D%0A%0D%0A%20%20%2F**%0D%0A%20%20%20*%20show%20the%20fraction%20of%20datapoint.x%2Fdatapoint.y%0D%0A%20%20%20*%2F%0D%0A%20%20fraction%20%3D%201%2C%0D%0A%0D%0A%20%20%2F**%0D%0A%20%20%20*%20show%20the%20percentage%20of%20(datapoint.x%2Fdatapoint.y)%25%0D%0A%20%20%20*%2F%0D%0A%20%20percentage%20%3D%202%0D%0A%7D%0D%0A%0D%0Aexport%20type%20ChartDataModeString%20%3D%20'default'%20%7C%20'fraction'%20%7C%20'percentage'%3B%0D%0A%0D%0Aconst%20testVar1%3A%20ChartDataMode%20%3D%20ChartDataMode.default%3B%0D%0A%0D%0Aconst%20testVar2%3A%20ChartDataModeString%20%3D%20'fraction') that shows the differences. Using an enum actually generates extra code where a string union usually suffices. All of the type code on a string union gets erased while still guaranteeing proper use.

Reply:
Change enum to string unions
type ChartDataMode = 'default' | 'fraction' | 'percentage';